clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -63,6 +63,12 @@ def err_drv_no_cuda_libdevice : Error<
   "via '--cuda-path', or pass '-nocudalib' to build without linking with "
   "libdevice">;
 
+def warn_function_fp_accuracy_already_set : Warning <"FP accuracy value of '%0' has already "
+  "been assigned to function '%1'">,
+  InGroup<FPAccuracyAlreadySet>;
+def warn_all_fp_accuracy_already_set : Warning <"FP accuracy value of '%0' has already "
+  "been assigned to all functions in the program">,
+  InGroup<FPAccuracyAlreadySet>;
```

Comment:
FP is fine in comments or discussion, but let's spell it out in user-facing contexts like diagnostics and documentation.

Also, these can be combined with a `%select`.

Reply:
The diag at line 69 is actually a left over from previous patch. Removed it.